recipes/phykit/meta.yaml

```diff
@@ -0,0 +1,265 @@
+    - pk_cs --help
+    - pk_faidx --help
+    - pk_get_entry --help
+    - pk_ge --help
+    - pk_gc_content --help
+    - pk_gc --help
+    - pk_pairwise_identity --help
+    - pk_pairwise_id --help
+    - pk_pi --help
+    - pk_parsimony_informative_sites --help
+    - pk_pis --help
+    - pk_relative_composition_variability --help
+    - pk_rel_comp_var --help
+    - pk_rcv --help
+    - pk_rename_fasta_entries --help
+    - pk_rename_fasta --help
+    - pk_sum_of_pairs_score --help
+    - pk_sops --help
+    - pk_sop --help
+    - pk_variable_sites --help
+    - pk_vs --help
+    - pk_bipartition_support_stats --help
+    - pk_bss --help
+    - pk_branch_length_multiplier --help
+    - pk_blm --help
+    - pk_collapse_branches --help
+    - pk_collapse --help
+    - pk_cb --help
+    - pk_covarying_evolutionary_rates --help
+    - pk_cover --help
+    - pk_dvmc --help
+    - pk_degree_of_violation_of_a_molecular_clock --help
+    - pk_evo_rate --help
+    - pk_evolutionary_rate --help
+    - pk_hidden_paralogy_check --help
+    - pk_clan_check --help
+    - pk_internal_branch_stats --help
+    - pk_ibs --help
+    - pk_internode_labeler --help
+    - pk_il --help
+    - pk_last_common_ancestor_subtree --help
+    - pk_lca_subtree --help
+    - pk_lb_score --help
+    - pk_long_branch_score --help
+    - pk_lbs --help
+    - pk_monophyly_check --help
+    - pk_is_monophyletic --help
+    - pk_nearest_neighbor_interchange --help
+    - pk_nni --help
+    - pk_patristic_distances --help
+    - pk_pd --help
+    - pk_polytomy_test --help
+    - pk_polyt_test --help
+    - pk_ptt --help
+    - pk_polyt --help
+    - pk_print_tree --help
+    - pk_print --help
+    - pk_pt --help
+    - pk_prune_tree --help
+    - pk_prune --help
+    - pk_rename_tree_tips --help
+    - pk_rename_tree --help
+    - pk_rename_tips --help
+    - pk_rf_distance --help
+    - pk_robinson_foulds_distance --help
+    - pk_rf_dist --help
+    - pk_rf --help
+    - pk_root_tree --help
+    - pk_root --help
+    - pk_rt --help
+    - pk_spurious_sequence --help
+    - pk_spurious_seq --help
+    - pk_ss --help
+    - pk_terminal_branch_stats --help
+    - pk_tbs --help
+    - pk_tip_labels --help
+    - pk_labels --help
+    - pk_tree_labels --help
+    - pk_tl --help
+    - pk_tip_to_tip_distance --help
+    - pk_t2t_dist --help
+    - pk_t2t --help
+    - pk_tip_to_tip_node_distance --help
+    - pk_t2t_node_dist --help
+    - pk_t2t_nd --help
+    - pk_total_tree_length --help
+    - pk_tree_len --help
+    - pk_treeness --help
+    - pk_tness --help
+    - pk_saturation --help
+    - pk_sat --help
+    - pk_treeness_over_rcv --help
+    - pk_toverr --help
+    - pk_tor --help
+    - pk_create_concatenation_matrix --help
+    - pk_create_concat --help
+    - pk_cc --help
+    - pk_thread_dna --help
+    - pk_pal2nal --help
+    - pk_p2n --help
+
+about:
+  home: "https://github.com/jlsteenwyk/phykit"
+  license: UNKNOWN
+  license_family: OTHER
+  license_file: 
```

Comment:
You should remove the unused fields: `license_family` (that is rarely set in Bioconda recipes anyway), `license_file`, `doc_url`, `dev_url`.